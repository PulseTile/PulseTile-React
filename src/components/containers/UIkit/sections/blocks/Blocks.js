import React from 'react';
import Header from '../fragments/blocks/Header';
import Footer from '../fragments/blocks/Footer';
import PatientInfo from '../fragments/blocks/PatientInfo';
import Breadcrumbs from '../fragments/blocks/Breadcrumbs';
import Sidebar from '../fragments/blocks/Sidebar';
import PanelWithFilter from '../fragments/blocks/PanelWithFilter';
import PatientSummaryDashboard from '../fragments/blocks/PatientSummaryDashboard';
import Vitals from '../fragments/blocks/Vitals/Vitals';
import Timeline from '../fragments/blocks/Timeline';
import BarChart from '../fragments/blocks/BarChart/BarChart';
import LineChart from '../fragments/blocks/LineChart';

/**
 * This component returns content of Blocks block of UIkit page
 *
 * @return {XML}
 * @constructor
 */
const Blocks = () => {
  return (
    <div id="blocks" className="ui-section">
      <h2 className="ui-main-title">Blocks</h2>
      <Header />
      <Footer />
      <PatientInfo />
      <Breadcrumbs />
      <Sidebar />
      <PanelWithFilter />
      <PatientSummaryDashboard />
      <Vitals />
      <Timeline />
      <BarChart />
      <LineChart />
    </div>
  );
};

export default Blocks;
