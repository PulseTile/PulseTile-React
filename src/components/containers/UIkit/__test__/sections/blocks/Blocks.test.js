import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Blocks from '../../../sections/blocks/Blocks';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Blocks />', () => {
  it('should renders Blocks with props correctly', () => {
    const component = shallow(<Blocks />);
    expect(component).toMatchSnapshot();
  });
});
