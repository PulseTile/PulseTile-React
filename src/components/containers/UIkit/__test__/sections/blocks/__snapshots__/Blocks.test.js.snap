// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Blocks /> should renders Blocks with props correctly 1`] = `
<div
  className="ui-section"
  id="blocks"
>
  <h2
    className="ui-main-title"
  >
    Blocks
  </h2>
  <Header />
  <Footer />
  <PatientInfo />
  <Breadcrumbs />
  <Sidebar />
  <PanelWithFilter />
  <PatientSummaryDashboard />
  <Vitals />
  <Timeline />
  <Connect(lifecycle(BarChart)) />
  <LineChart />
</div>
`;
